sdk/src/feature_set.rs

```diff
@@ -665,6 +665,11 @@ pub mod checked_arithmetic_in_fee_validation {
     solana_sdk::declare_id!("5Pecy6ie6XGm22pc9d4P9W5c31BugcFBuy6hsP2zkETv");
 }
 
+pub mod remaining_compute_units_syscall_enabled {
+    // TODO: replace by team
+    solana_sdk::declare_id!("GH4auHBrmun5EYthnRw8sQ6JZhGWGsUJNPzuNGKEq37C");
```

Comment:
Here's the pubkey for a keypair that I hold
```suggestion
    solana_sdk::declare_id!("5TuppMutoyzhUSfuYdhgzD47F92GL1g89KpCZQKqedxP");
```